pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>io.cdap.plugin</groupId>
   <artifactId>amazon-s3-plugins</artifactId>
-  <version>1.17.3</version>
+  <version>1.17.4</version>
```

Comment:
Please take this change separately.

Reply:
+1, we should change it to `1.17.4-SNAPSHOT` as part of this PR.

Reply:
Updated to `1.17.4-SNAPSHOT` as part of this PR. 79e1df818f856438b79b4431b49d12d8ceb8811c
Will raise a separate PR for removing the `-SNAPSHOT`